cloud/deploys/staging_azure/README.md

```diff
@@ -154,4 +161,13 @@ To do that add the custom records via the domain provider webiste.
 1) CNAME record which points from the 'staging-azure.civiform.dev' to the hostname that gets generated from terraform (you can find this from the terraform output or via the azure portal)
 2) TXT record with key 'asuid.staging-azure.civiform.dev' and value that matches the custom domain verification id in the azure portal (you can find this by navigating to the custom domains in the app service setting). 
 
-Note it should take a few minutes to propagate.
\ No newline at end of file
+Note it should take a few minutes to propagate.
+
+# Configure AWS sending 
+You will need an aws key/secret that you can get from the AWS console. The key is specified via terraform and the secret lives in the azure secrets store. 
```

Comment:
suggestion (blocking): Can you add a CLI command to add this secret to the key vault in the Key Vault Setup section? you can reorder the sections if it makes sense. Eventually this step will be automated so the key vault setup script will prompt for this secret in the terminal but for now it might be useful to document this step.

Reply:
will do in a follow up PR

Reply:
#1996